provider/openstack/provider.go

```diff
@@ -1338,6 +1338,19 @@ func (e *Environ) startInstance(
 	}, nil
 }
 
+func (e *Environ) userFriendlyInvalidNetworkError(err error) error {
+	msg := fmt.Sprintf("%s\n\t%s\n\t%s", err,
```

Comment:
You probably want `err.Error()` as the Sprintf argument just in case the passed error type does not implement `fmt.Stringer`

Reply:
done